### ISSUE TITLE
TensorFlow accuracy

### ISSUE THREAD

**Issue by RK900** [Eternagame/EternaBrain#16]
- Change learning rate (below 0.0001)
- Change dropout rate (below 0.5)
- Increase number of epochs
- Change number of layers
- Change number of nodes
